build/Build.cs

```diff
@@ -140,6 +140,13 @@ class Build : NukeBuild
     Target PrepareDevelopmentEnvironment => _ => _
         .Executes(() =>
         {
+            SourceDirectory.GlobFiles("**/WebResources/*/package.json").ForEach(packageFile =>
+            {
+                NpmTasks.NpmRun(s => s
+                    .SetProcessWorkingDirectory(packageFile.Parent)
+                    .SetCommand("build"));
+            });
+
             InstallSolutionAndDependencies(DataverseSolution, SolutionType.Unmanaged);
         });
```

Comment:
Thanks, @mjahlv. I've just had a review of the Build.cs file and this was definitely an oversight. I think it could even a bit more concise - 

```suggestion
    Target PrepareDevelopmentEnvironment => _ => _
        .DependsOn(Compile)
        .Executes(() =>
        {
            InstallSolutionAndDependencies(DataverseSolution, SolutionType.Unmanaged);
        });
```

Reply:
I think this would also mean we could delete these lines from `InstallSolutionAndDependencies`

```csharp
        DotNetBuild(s => s
            .SetProjectFile(SolutionsDirectory / solution / $"{solution}.cdsproj")
            .SetConfiguration(buildConfiguration)
            .SetDisableParallel(true));
```

Reply:
@ewingjm that works! I've just pushed in the change :)